### ISSUE TITLE
SPM support for macOS?

### ISSUE THREAD

**Issue by piers12** [mixpanel/mixpanel-swift#420]
Any idea when you plan to provide support for macOS? is It currently a work in progress?

**Comment by zihejia** [comment]
Hi @piers12 , it's in our backlog. We will let you know when we start working on it. Thanks for raising it.

**Comment by HelgeBecker** [comment]
What is the current state on this issue? (just closing it is not a solution)

**Comment by zihejia** [comment]
Hi @HelgeBecker , @piers12 , sorry for the waiting. We've now added SPM support to our [3.0.0.beta](https://github.com/mixpanel/mixpanel-swift/releases/tag/v3.0.0.beta.2) release for macOS. 
```
1.  In Xcode, select File > Swift Packages > Add Package Dependency.
2.  Follow the prompts using the URL for this repository and point to the branch `3.0.0.beta`
```
Here is more context:
On Jan 1, 2022, we’ll remove the [Messages & Experiments(M&E)](https://mixpanel.com/blog/why-were-sunsetting-messaging-and-experiments/#:~:text=A%20year%20from%20now%2C%20on,offering%20discounts%20for%20getting%20started) feature from Mixpanel. If you are not using messages or A/B testing, you can definitely use this beta version and it's in 100% parity with the main version excluding the M&E feature.

Going forward, we don't have a plan to add macOS SPM support to the 2.x.x release. It will only be available to 3.0.0+.  I'm closing this issue now, please feel free to reopen if there are issues on the 3.0.0.beta branch for the macOS SPM support
